hyperspy/roi.py

```diff
@@ -409,7 +409,7 @@ def interactive(self, signal, navigation_signal="same", out=None,
         Returns
         -------
         :py:class:`~hyperspy.signal.BaseSignal` or one of its subclass
-            Signal updated with the current ROI selection 
+            Signal updated with the current ROI selection
```

Comment:
This is the kind of changes, which is not great for history but if this is pain to remove, we can keep it.

Reply:
Not sure if it was the best way to do it but I created a new commit undoing the change and then squashed the two commits... Seems like it worked.